jujugui/static/gui/src/app/components/profile/navigation/_navigation.scss

```diff
@@ -1,30 +1,30 @@
 .profile-navigation {
   color: $color-dark;
+  padding: 0 1rem;
 
-  &__list-item {
-    list-style: none;
+  .p-list__item {
     cursor: pointer;
+    list-style: none;
 
     &:nth-child(4) {
-      border-top: 1px solid #cdcdcd;
-      padding-top: 12px;
-      margin-right: 24px;
+      border-top: 1px solid $color-mid-light;
+      margin-top: 1rem;
+      padding-top: 1rem !important;
```

Comment:
I think adding an element to the list such as: `<li class="p-list__item"><hr class="u-no-margin--bottom"></li>` is clearer then tarenting `nth-child` in css.

Reply:
This was my preferred approach also but as these links are generated from a section map object, that would involve splitting the section map and populating two different lists etc which is, in my opinion, adding too much complexity just to accommodate a stylistic keyline. 

However, relying on `nth:child` is rather brittle so as an improvement I have instead added a unique class to each list item and refactored the SCSS to target the `Cloud Credentials` item instead.